src/features/smartSearch/components/filters/PersonTags/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { FormEvent } from 'react';
+import { FormEvent, useState } from 'react';
 import { Box, Chip, MenuItem } from '@mui/material';
```

Comment:
This causes the lint error

Reply:
I'll resolve this, no problem! Don't know why my own linting hook did not catch it.